models/Habit.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+// Create Habit schema
+const habitSchema = new Schema({
```

Comment:
I think that it would be useful to have something like `dateCreated` so that we have some means to sort this habits if needed.

Reply:
Good idea